hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/utils/TenantConfigHelper.java

```diff
@@ -43,11 +45,21 @@ public static TenantConfigHelper usingContext(final SystemSecurityContext system
 
     /**
      * Is multi-assignments enabled for the current tenant
-     * 
+     *
      * @return is active
      */
     public boolean isMultiAssignmentsEnabled() {
         return systemSecurityContext.runAsSystem(() -> tenantConfigurationManagement
                 .getConfigurationValue(MULTI_ASSIGNMENTS_ENABLED, Boolean.class).getValue());
     }
+
+    /**
+     * Returns the wanted configuration
+     *
+     * @return configuration value
+     */
+    public <T extends Serializable> T getConfigValue(final String key, final Class<T> valueType) {
```

Comment:
This only seems to be used once to get the multiassignment default weight. Maybe making it private and use it in isMultiAssignmentsEnabled and offer a public getMultiAssignmentsSefaultWeight would be better. That way we would not offer a public generic method that runs with system privileges, which might be called by mistake. (Maybe also name it getConfigValueWithSystemPrivileges)